Sources/DataPipeline/Config/SDKConfigBuilder.swift

```diff
@@ -7,10 +7,11 @@ import Segment
 /// the SDK, ensuring all required and optional settings are appropriately configured before the
 /// SDK is initialized.
 ///
-/// Usage Example:
+/// **Usage Example:**
+///
 /// ```
 /// let config = SDKConfigBuilder(cdpApiKey: "your_cdp_api_key")
-///   .apiHost("your_api_host")
```

Comment:
wonder why we removed this? 🤔

Reply:
I just tried updating example with more common usage 😄